R/keep.R

```diff
@@ -155,14 +156,25 @@ keep_podiatrist_services <- function(sysi, sssy) {
     dplyr::filter(!!logic) |>
     # Remove duplicates
     dplyr::distinct() |>
-    # Keep only the two columns we need and transform `honuge` to YYYY-MM-DD.
-    dplyr::mutate(
-      pnr = .data$pnr,
-      date = yyww_to_yyyymmdd(.data$honuge),
-      .keep = "none"
+    # Keep only the two columns we need.
+    dplyr::select(
+      pnr = "pnr",
+      date = "honuge"
     ) |>
     # Add logical helper variable to indicate diabetes-related podiatrist service.
     dplyr::mutate(from_podiatrist_service = TRUE)
+
+  podiatrist_services |>
+    # Transform `honuge` to YYYY-MM-DD.
+    # Unfortunately, the conversion from YYWW to an actual date is quite tricky
+    # to do well in SQL without a lot of revising and refactoring the code.
+    # So instead, we'll just pull into R, convert, and then bring back
+    # to DuckDB.
+    # TODO: Fix to use DuckDB SQL only.
```

Comment:
This was the only way to get past this point, was to pull into R and than convert back into DuckDB.